.github/workflows/ci.yml

```diff
@@ -8,18 +8,26 @@ jobs:
     strategy:
       max-parallel: 1
       matrix:
-        ruby: ['2.5', '2.6', '2.7', '3.0']
-    name: Ruby ${{ matrix.ruby }}
+        ruby: ['2.5', '2.6', '2.7', '3.0', '3.1']
+    name: 💎 Ruby ${{ matrix.ruby }}
     steps:
       - uses: actions/checkout@v2
+        with:
+          fetch-depth: 0 # gives the commit linter access to previous commits
+
+      - name: Commit message linter
+        if: ${{ matrix.ruby == '2.5' }}
```

Comment:
why to run in old version? since it's the first one?

Reply:
@ferhatelmas it's the first one that runs in the matrix

Reply:
thanks